Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM python:3.6-slim
+COPY setup.* /opt/airflow/
+COPY airflow /opt/airflow/airflow
+
+ARG AIRFLOW_HOME=/usr/local/airflow
+ARG AIRFLOW_DEPS="all"
+ARG PYTHON_DEPS=""
+ARG buildDeps="freetds-dev libkrb5-dev libsasl2-dev libssl-dev libffi-dev libpq-dev git"
+ARG APT_DEPS="$buildDeps libsasl2-dev freetds-bin build-essential default-libmysqlclient-dev apt-utils curl rsync netcat locales"
+
+RUN set -x \
+    && export SLUGIFY_USES_TEXT_UNIDECODE=yes \
+    && apt update \
+    && if [ -n "${APT_DEPS}" ]; then apt install -y $APT_DEPS; fi \
+    && if [ -n "${PYTHON_DEPS}" ]; then pip install ${PYTHON_DEPS}; fi \
```

Comment:
Hej @mik-laj. You are right if this is something done in a single RUN call of the Dockerfile. But if we decide to go the direction I proposed in https://github.com/ffinfo/incubator-airflow/pull/1, and have multiple layers of the image, it makes sense to keep the cache dir because there you have subsequent calls to pip install - both using the same cache. 

In the solution I proposed in my pull requests, you run second 'pip install' of airflow sources after installing dependencies - which will result in optimised experience of users - they will only download/build incremental layers. Without it, the whole image will have to be downloaded /rebuilt including all dependencies after every single change to airflow sources.

Reply:
I now did add `--no-cach-dir`. This did make the image 150Mb smaller 👍